### PR TITLE
UI updates

### PR DESCRIPTION
**Pull request by gwil**
Updates to the following ui issues:

- Intro and outro animations on hover/player state
- Adds "distance from thumb" to offset position
- Progress indicator position and animation
- Prevents scrolling on iOS swipe state
- Fix volume icon during hover/swipe mode and during toggle
- Fonts on iOS
- Formats view count

### CODE REVIEW

File: dist/index.html [hunk 1]
```diff
@@ -0,0 +1,46 @@
+<!DOCTYPE html>
```
Comment (eudisd) [inline dist/index.html]: I think @lukeed removed this from the repo and it's auto generated now no?

File: src/helpers/__tests__/utils.spec.js [hunk 2]
```diff
@@ -306,29 +306,30 @@ describe('Util Tests', () => {
...
-
-				const output = util.setPlayerProfilePosition(player, profile, event);
-				expect(output).toBe(undefined);
-
-				expect(player.style.transform).toBe(transform);
-				expect(profile.style.transform).toBe(transform);
-			});
-		});
-	})
+	// for now at least on Social Vidy's, the profile is no longer attached to the player
```
Comment (eudisd) [inline src/helpers/__tests__/utils.spec.js]: I recommend we remove this if we don't need to test against it anymore

File: src/helpers/util.js [hunk 3]
```diff
@@ -54,13 +54,14 @@ export function toLocation(str, sub, n) {
 }
 
 export function getMobilePlayerPosition(player, event, isExpandedView) {
+	const touchDistance = 40; // space for finger
```
Comment (eudisd) [inline src/helpers/util.js]: 👏

File: src/helpers/util.js [hunk 4]
```diff
@@ -94,10 +95,27 @@ export function getMobilePlayerPosition(player, event, isExpandedView) {
...
 }
 
 export function setPlayerProfilePosition(player, profile, event, isExpandedView) {
 	const position = getMobilePlayerPosition(player, event, isExpandedView);
-	player.style.transform = profile.style.transform = position;
+	// player.style.transform = profile.style.transform = position;
+	player.style.transform = position;
+}
+
+export function toCountSuffix(n) {
```
Comment (eudisd) [inline src/helpers/util.js]: Not required, If you're feeling wild 😄, we can add a test case for this.  I can show you how with the awesomeness that is `Jest`

File: dist/index.html (hunk 1 above)
Comment (lukeed) [reply dist/index.html]: It's not autogenerated; it just lives in `discards` directory and is copied over for development. The last PR (#21) addressed this.

Comment (eudisd) [reply dist/index.html]: Ah got it, my bad

Comment (lukeed) [reply dist/index.html]: Lol no worries

File: src/helpers/util.js (hunk 4 above)
Comment (gwil) [reply src/helpers/util.js]: Added a test for `toCountSuffix`

Comment (eudisd) [reply src/helpers/util.js]: NICE!

Comment (eudisd) [reply src/helpers/util.js]: Thanks!

**Comment by eudisd** [comment]
Going to check it out locally on my iphone/android, will let you know if I find any issues

**Pull request merged**